pallets/task/src/lib.rs

```diff
@@ -217,6 +231,15 @@ pub mod pallet {
 	#[pallet::generate_store(pub(super) trait Store)]
 	pub struct Pallet<T>(_);
 
+	//TODO!!!!!!! BOUNDED VEC SIZE
+	#[pallet::storage]
+	/// Tasks: The Tasks that end on a given block number [key: Block Number, value: Vec<TaskId>]
+	pub(super) type ExpiringTasksPerBlock<T: Config> = StorageMap<_, Twox64Concat, T::BlockNumber, BoundedVec<T::Hash, MaximumTasksPerBlock>, ValueQuery>;
```

Comment:
Arent these 2 storage items the same? what is the difference between Expiring and DyingTasks?